### PR TITLE
US9852 - add user different leader email

### PR DESCRIPTION
**Pull request by plachmann**

### CODE REVIEW

File: Gateway/crds-angular/Controllers/API/FinderController.cs [hunk 1]
```diff
@@ -30,7 +29,7 @@ public class FinderController : MPAuth
         private readonly IAuthenticationRepository _authenticationRepo;
         private readonly IAnalyticsService _analyticsService;
         private readonly ILog _logger = LogManager.GetLogger(MethodBase.GetCurrentMethod().DeclaringType);
-        private const int _trialMemberRoleId = 67;
+        private const int TrialMemberRoleId = 67;
```
Comment (JoeKer1) [inline Gateway/crds-angular/Controllers/API/FinderController.cs]: Should that number go in the MinistryPlatform config thing?

Comment (JoeKer1) [reply Gateway/crds-angular/Controllers/API/FinderController.cs]: Actually I think it's already there should we get it from there in the 0% chance we change the id?

File: Gateway/crds-angular/Services/FinderService.cs [hunk 2]
```diff
@@ -1223,53 +1282,6 @@ public void SendEmailToAddedUser(string token, User user, int groupid)
             RecordCommunication(connection);
         }
        
-        private List<PinDto> TransformGroupDtoToPinDto(List<GroupDTO> groupDTOs, string finderType)
```
Comment (JoeKer1) [inline Gateway/crds-angular/Services/FinderService.cs]: did we not use this?

Comment (plachmann) [reply Gateway/crds-angular/Services/FinderService.cs]: VS showed it as unused so I removed it.

**Pull request merged**